docs/how-tos/adding-events-to-event-bus.rst

```diff
@@ -0,0 +1,6 @@
+How to add an Open edX Event to the Event Bus
+=============================================
+
+Open edX Events can be sent synchronously in the same Django process where the
+service runs or asynchronously using the event bus mechanism. We've put together
+this guide on how to send new events over the event bus.
```

Comment:
@robrap @rgraber: I was checking [this confluence](https://openedx.atlassian.net/wiki/spaces/AC/pages/3508699151/How+to+start+using+the+Event+Bus) page to add it here as part of the documentation. But then I wandered, does this page change too much? Should I link that confluence page while it gets more stable? What do you think?

Reply:
Linking to this doc would be great. It may change, but I think it is a stable landing page for working with the event bus.

Reply:
good! thank you